tests/fox_scraper_tests.py

```diff
@@ -1,12 +1,81 @@
 from nose.tools import *
-import fox_scraper
+from fox_scraper import Scraper
+from fox_scraper import FoxItem
+from bs4 import BeautifulSoup
+from tests.items_src import catalog
 
```

Comment:
because this is how the tutorial showed (https://learnpythonthehardway.org/book/ex47.html)
this is how it works with the nosetests.